index.js

```diff
@@ -4,17 +4,23 @@ const fs = require('fs')
 const tablemark = require('tablemark')
 const isValidPath = require('is-valid-path')
 
+var jsonIsArray = /^\s*\[/
```

Comment:
These should be `const`

Reply:
Also, if you sense inconsistency with my `let / const` usages, I was doing a 'prefer-let' thing for a while but I'm moving away from that. Realized it was a rebellion against `const` being 5 chars while `let` & `var` were 3 :smile: I'm back to `const` for pretty much everything and `let` when the value will change.

Reply:
Ah, missed that. I have indeed been unable to determine your preference on that. I've personally settled on the same as you describe: `const` always, unless `let` is necessary.